logstash-core/src/test/java/org/logstash/config/ir/RubyEnvTestCase.java

```diff
@@ -44,15 +42,14 @@ public static void before() {
     private static void ensureLoadpath() {
         final LoadService loader = RubyUtil.RUBY.getLoadService();
         if (loader.findFileForLoad("logstash/compiler").library == null) {
+            final String gems = LS_HOME.
```

Comment:
it seems a variation of our `safePath` helper from elsewhere may come in handy here.

Reply:
honestly - this whole piece could use a refactoring as it's a bit worrying.
would rather drop it (had issues with this running against jruby-head) - but decided it's out-of-scope here.
there isn't anything new here - the `vendor/bundle/jruby` wasn't validated before ...
what's new is that we no longer have to rely on a system property being set to resolve the path.